calculator/calculator/CalculatorDataSheet.swift

```diff
@@ -0,0 +1,37 @@
+//
+//  MemoData.swift
+//  memoApp (iOS)
+//
+//  Created by Byeon jinha on 2022/04/30.
+//
+
+import Foundation
+
+final class CalculatorDataSheet : ObservableObject{
+    @Published var viewNumS : String = "0" {
+        didSet {
+            viewNumC = Array(viewNumS)
+        }
+    }
+    @Published var viewNumS2 : String = "0" {
+        didSet {
+            viewNumC2 = Array(viewNumS2)
+        }
+    }
+    @Published var viewNum : Double = 0 {
+        didSet {
+            viewNumS = String(viewNum)
+        }
+    }
+    @Published var isPlus : Bool = false 
+    @Published var isMinus : Bool = false
+    @Published var isMultiply : Bool = false
+    @Published var isDivide : Bool = false
+    
+    @Published var isCalculation : Bool = false
+    @Published var isFirstCalculation : Bool = false
+    @Published var isAC : Bool = false
+    @Published var isActCalculation : String = ""
+    @Published var viewNumC : [Character] = ["0"]
+    @Published var viewNumC2 : [Character] = ["0"]
```

Comment:
```suggestion
    @Published var viewNumS = "0" {
        didSet {
            viewNumC = Array(viewNumS)
        }
    }
    @Published var viewNumS2 = "0" {
        didSet {
            viewNumC2 = Array(viewNumS2)
        }
    }
    @Published var viewNum : Double = 0 {
        didSet {
            viewNumS = String(viewNum)
        }
    }
    @Published var isPlus = false 
    @Published var isMinus = false
    @Published var isMultiply = false
    @Published var isDivide = false
    
    @Published var isCalculation = false
    @Published var isFirstCalculation = false
    @Published var isAC = false
    @Published var isActCalculation = ""
    @Published var viewNumC : [Character] = ["0"]
    @Published var viewNumC2 : [Character] = ["0"]
```

Swift는 자동으로 타입추론이 되기 때문에 Double 처럼 CGFloat 같은 경우 Compile시, 혼란이 생기는 경우 이외에는 생략을 추천합니다 : )

Reply:
오오... 알겠습니다! 생략하는 걸 연습하겠습니다. 그리고 수정 완료했습니다. :)